src/pages/home/report/ReportActionItem.js

```diff
@@ -199,9 +202,10 @@ function ReportActionItem(props) {
 
     const toggleReaction = useCallback(
         (emoji) => {
-            Report.toggleEmojiReaction(props.report.reportID, props.action.reportActionID, emoji);
+            Report.toggleEmojiReaction(props.report.reportID, props.action, emoji, props.emojiReactions);
+            Report.toggleEmojiReaction(props.report.reportID, props.action.reportActionID, emoji, props.emojiReactions);
```

Comment:
I don't think we need to call `toggleEmojiReaction` twice here?

Reply:
Correct. Bad merge